x/commerciodocs/keeper.go

```diff
@@ -71,7 +71,7 @@ func (keeper Keeper) addReaderForDocument(ctx sdk.Context, user types.Did, refer
 		keeper.cdc.MustUnmarshalBinaryBare(existingReaders, &readers)
 	}
 
-	// Append the user to the reading list
+	// Append the ownerIdentity to the reading list
```

Comment:
I think this might be a refactoring error. Could we bring `ownerIdentity` back to `user`?